index.html

```diff
@@ -350,7 +350,7 @@ <h2 id="scope"><span class=secno>3 </span>Scope</h2>
   <p>This specification is limited to providing DOM events for retrieving
   information describing the physical orientation and motion of the hosting
   device. The intended purpose of this API is to enable simple use cases such
-  as those in <a href="#usecases">Section 5.2</a>.
+  as those in <a href="#usecases">Section 6.1</a>.
```

Comment:
Good catch! The right fragment identifier seems to be `#use-cases`, if you fix that this is good to be merged.

https://w3c.github.io/deviceorientation/#use-cases

Reply:
Fixed!

Reply:
Thanks!